tests/test_tutorials.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+cd ../tutorials
+g++ -std=c++11 -march=native -o3 lattice_base_generation_tutorial.cpp -lfplll -lmpfr -lgmp -o test_base
+g++ -std=c++11 -march=native -o3 lll_reduction_tutorial.cpp -lfplll -lmpfr -lgmp -o test__lll_reduction
+g++ -std=c++11 -march=native -o3 bkz_reduction_tutorial.cpp -lfplll -lmpfr -lgmp -o test_bkz_reduction
+mv test_base ../tests/lattices
```

Comment:
All of this should happen in a temporary directory using `mktemp` instead of in the source tree.